pkg/bundle/client.go

```diff
@@ -85,7 +85,18 @@ func (bc *bundleClient) GetPackageBundleController(ctx context.Context) (*api.Pa
 	})
 	err := bc.List(ctx, u)
 	if err != nil {
```

Comment:
Just log this error as a warning and remove the rest of the code. Line 103 `name := api.PackageBundleControllerName` will default it.

Reply:
But this will not require the user to modify their pbcs, that's why I added the listing of the pbcs that currently exist in the cluster. Currently, there will only be 1.